saleor/graphql/order/mutations/draft_orders.py

```diff
@@ -360,6 +360,18 @@ class Meta:
         error_type_class = OrderError
         error_type_field = "order_errors"
 
+    @classmethod
+    def clean_instance(cls, info, instance):
```

Comment:
It would be good to add tests for this method.

Reply:
It's tested in `test_draft_order_delete_non_draft_order`.

Reply:
Not directly but agreed :)